### ISSUE TITLE
Betterdiscord not opening.

### ISSUE THREAD

**Issue by itsvara** [Jiiks/BetterDiscordApp#864]
Whenever I try to boot up discord, it boots up fine, but betterdiscord doesn't load up, I tried waiting for 20 minutes, I tried restarting it, I restarted it using task manager, I even logged out and logged back in, nothing works.

**Comment by M-archand** [comment]
Same, BandagedBD isn't fixing it for me either like it usually does.

**Comment by AyanamiRei1** [comment]
Oh this is isn't too good. oh well I'll just wait until it's fixed either in the Bandaged fork or this fork.

**Comment by intrnl** [comment]
Have you tried *reinstalling*?

**Comment by AyanamiRei1** [comment]
Yep just did that can it's working just fine.

**Comment by BetterDiscordFan** [comment]
Well I prefer reinstalling BetterDiscord, because it works that way. You also need to go to WINDOWS KEY + R or type in your bottom left search bar "Run". Then a little run menu will popup, then type in the run box and put "%appdata%. After, you should see a BetterDiscord folder, you can keep your plugins in a different folder to save them or just find them in your way. Then delete the folder, after that you must reinstall the bandagebd. After, you are good to go :) .

**Comment by Defty101** [comment]
for me a running the bandaged BD installer again fixed it

**Comment by intrnl** [comment]
considering that the person above you stated that it is working, then it is working, because it is in fact working, betterdiscord isn't broken, it still works.

**Comment by Zapiboss** [comment]
Reinstalling Betterdiscord didn't worked for me.
But, installing bandageddiscord instead, did the trick and i even can still my my plugins.

**Comment by intrnl** [comment]
You're supposed to install BandagedBD anyway.

**Comment by flamefx** [comment]
Hey, I have been having some issues with this too. I backed up the file I thought were important, plugins, themes, bdsettings.json and bstorage.json.
on reinstalling, i first checked that it worked alone (it did),
then I added all my backup files - didn't work.
so I removed them once more - it worked. 
the next time I only added plugins and themes - it worked. next bdsettings - it worked.
finally i added bstorage - didn't work.

Now that I knew MY issue lied with the bdstorage, i decided to open my backed up version and check on its contents - turns out it was empty. Then on further inspection of the one I had already this seems to be the format: 
```
{
    "defaults": {
        "data": {}
    },
    "data": {},
    "plugin Name": "{\"version\":\"XX\"}"
}
```
so if you are experiencing issues with this - I urge you to try to remove the `bdstorage.json`
file to see if that fixes anything.
